pkg/testskeleton/testskeleton.go

```diff
@@ -58,47 +58,25 @@ type AdditionalChangesAfterDeployment struct {
 	ChangedResources     []ChangedResource
 }
 
-// Structure used for AWS deployments - contains randomly generated resource names.
-type AwsRandomNames struct {
-	NamePrefix string
+// Structure used for terraform tfvars input information,
+// not all options are used in all public cloud environments
+type TerraformVarsInfo struct {
+	NamePrefix              string
+	AzureResourceGroupName  string
+	AzureStorageAccountName string
+	GCPProjectID            string
 }
 
-// Function that generates and returns a set of random AWS resource names.
-// Randomization is based on UUID.
-func GenerateAwRandomNames() AwsRandomNames {
+// Function that generates and returns information used by Terraform TFVARS.
+func GenerateTerraformVarsInfo() TerraformVarsInfo {
 	prid := os.Getenv("PRID")
 	if prid != "" {
 		prid = fmt.Sprintf("p%s", prid)
 	} else {
 		prid = "tt"
 	}
 
-	id := uuid.New().String()
-	idSliced := strings.Split(id, "-")
-
-	prefixId := idSliced[2]
-
-	names := AwsRandomNames{
-		NamePrefix: fmt.Sprintf("%s-%s-", prid, prefixId),
-	}
-
-	return names
-}
-
-// Structure used for Azure deployments - contains randomly generated resource names.
-type AzureRandomNames struct {
-	NamePrefix         string
-	ResourceGroupName  string
-	StorageAccountName string
-}
-
-// Function that generates and returns a set of random Azure resource names.
-// Randomization is based on UUID.
-func GenerateAzureRandomNames() AzureRandomNames {
-	prid := os.Getenv("PRID")
-	if prid != "" {
-		prid = fmt.Sprintf("-pr%s-", prid)
-	}
+	project_id := os.Getenv("PROJECT_ID")
```

Comment:
Can we use [camel case for naming variables](https://betterprogramming.pub/naming-conventions-in-go-short-but-descriptive-1fa7c6d2f32a) ? e.g.  

```suggestion
	projectId := os.Getenv("PROJECT_ID")
```